### DISCUSSION TITLE
Question about Placeholder Styles in Windows High Contrast Mode.

### DISCUSSION THREAD

**Discussion by TakahiroHimi** [mdn/mdn-community#484] [Content]
Thank you always for managing such wonderful content.

I have a question regarding [the section about placeholder styles in Windows High Contrast Mode](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder#windows_high_contrast_mode), found in your content on placeholders. The content states that "Placeholder text will appear with the same styling as user-entered text content when rendered in [Windows High Contrast Mode](https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/)." However, when I compared the actual input values and the placeholder in Windows 10's High Contrast Mode, they appeared in different styles, as shown in the attached image (left is the actual input, right is the placeholder).

Since the style of the placeholder seems to be considered in Windows High Contrast Mode, it might be better to remove the section on Windows High Contrast Mode. Would it be okay if I create a Pull Request for its removal?

![image](https://github.com/mdn/content/assets/60545096/6149fbe2-c2fe-449c-9a83-64872e7c43c4)

**Comment by Rumyra** [comment]
Hi @TakahiroHimi thanks for bringing this up - can you provide a link to the codepen for which you have a screenshot - thank you!

**Reply by TakahiroHimi** [reply]
@Rumyra 
Thank you for your reply. Here is the link to CodePen.
https://codepen.io/Takahiro-HIMI/full/ZEwPqPe
